packages/api/src/command/medical/document/document-redownload.ts

```diff
@@ -293,6 +297,11 @@ export const fhirContainedToCW = (contained: Resource): Contained | undefined =>
         identifier: contained.identifier?.map(fhirIdentifierToCW),
         name: contained.name,
       };
+    case "Practitioner":
+      return {
+        ...base,
+        identifier: contained.identifier?.map(fhirIdentifierToCW),
+      };
```

Comment:
This is not ideal, but this code is only used when we call `POST /internal/docs/reprocess` w/o the `re-query-doc-refs`, which is not the case now.

Ideally, we'd do one of the items recommended on the `TODO` at the top of the function - either:
- remove this (and we don't convert doc refs from FHIR to CW); or
- make this and caller better match the CW>FHIR conversion (on `toFHIR()`).

Both will take time and I'd rather push this through to deliver #846 sooner than later. Maybe we could disable the possibility to call `POST /internal/docs/reprocess` w/o the `re-query-doc-refs`?

Reply:
Should we push this as is and raise a ticket for the TODO to be addressed at a later time?

Reply:
https://github.com/metriport/metriport-internal/issues/1025